ci-script-windows.sh

```diff
@@ -56,13 +56,8 @@ execute(){
 build_darktable() {
     cd "$(dirname "$0")"
 
-    # for RawSpeed submodule
-    git submodule init
-    git submodule update
-
-    mkdir artifacts
     mkdir build && cd build
-    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/artifacts ../.
+    cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/c/projects/darktable/build ../.
```

Comment:
At least for me ninja has some problems with building dt on Windows: It opens some popups, asking about how to open .sh files (scripts in tools/ we run during compilation).